### ISSUE TITLE
Implement `kontrol build` tool

### ISSUE THREAD

**Issue by ehildenb** [runtimeverification/kontrol#2]
A new python utility (using our python project template) that implements `kontrol build`.

It should:

- Detect if it's a Foundry/Hardhat/Truffle project.
- If it's a Foundry project, call `forge build ...` with the extra options we need: https://github.com/runtimeverification/evm-semantics/blob/master/tests/foundry/foundry.toml#L4
- If it's a Hardhat/Truffle project, it should say "Unimplemented, go to https://kontrol.runtimeverification.com to request feature."

**Comment by yale-vinson** [comment]
@tothtamas28 is this still blocked by #1649, or can we proceed with this from your perspective?

**Comment by tothtamas28** [comment]
> is this still blocked by #1649

It is not. This issue subsumes https://github.com/runtimeverification/kontrol/issues/34 though. Also, Truffle support is probably no longer in scope.

**Comment by yale-vinson** [comment]
Closed as a duplicate of [#34](https://github.com/runtimeverification/kontrol/issues/34) as discussed in the Standup on 26 Sep.
